common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/Utils.java

```diff
@@ -23,4 +24,22 @@ public static void copyResultFields(MojoFrame mojoFrame, List<Row> outputRows) {
       }
     }
   }
+
+  /**
+   * Sanitize boolean string literal values true / false (case insensitive) into 1 / 0 respectively.
+   * @return sanitized string.
+   */
+  public static String sanitizeBoolean(String value, DataField.DataTypeEnum dataType) {
+    if (
+        dataType.equals(DataField.DataTypeEnum.FLOAT32)
+            || dataType.equals(DataField.DataTypeEnum.FLOAT64)
+    ) {
+      if (value.matches("(?i)true")) {
```

Comment:
nit: I would go for `if ("true".equalsIgnoreCase(value)) {` for safe handling of null values and avoid using inline regexes.

Reply:
good catch ! thx @rasika , wonder why i did not come up with this in the first place.. updated https://github.com/h2oai/dai-deployment-templates/pull/327/commits/ece8b620f6342a4f70e1c20a32c368ee06738b57